extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -97,6 +102,22 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 						name="email"
 					/>
 				</p>
+				<?php foreach ( $values['interests'] as $interest ) : ?>
+					<input
+						name="interests[<?php echo esc_attr( $interest ); ?>]"
+						type="hidden"
+						class="mc-submit-param"
+						value=1
```

Comment:
Do no quotes turn input value to a Number instead of String? :-)

Reply:
In this case not sure it matters since all we need is a truthy value, but might as well be explicit so I added quotes: https://github.com/Automattic/jetpack/pull/14050/commits/c8850868b923f00e1633c43beea1def6ce48c2ab